atom/common/api/atom_api_native_image.cc

```diff
@@ -350,6 +359,7 @@ void NativeImage::BuildPrototype(
   mate::ObjectTemplateBuilder(isolate, prototype)
       .SetMethod("toPNG", &NativeImage::ToPNG)
       .SetMethod("toJPEG", &NativeImage::ToJPEG)
+      .SetMethod("toBUFFER", &NativeImage::ToRawBuffer)
```

Comment:
We should probably use `toBitmap` as name, since other APIs are also returning `Buffer`s.